Sources/Extensions/UIKitExtension.swift

```diff
@@ -71,10 +71,13 @@ public extension UITableView {
         for changeset in stagedChangeset {
             if let interrupt = interrupt, interrupt(changeset), let data = stagedChangeset.last?.data {
                 setData(data)
-                return reloadData()
+                reloadData()
+                return
             }
+        }
 
-            _performBatchUpdates {
+        _performBatchUpdates {
+            for changeset in stagedChangeset {
```

Comment:
This applies all changes in one single `performBatchUpdates`, is that intentional? There is a reason updates are split into multiple `performBatchUpdates`.

Reply:
Yes, this is intentional. After my research only this change has helped me. I did it some time ago, so I do not remember all the details, but idea behind it: if we have multiple context for update, sometimes there are situation when we remove some cells in one context, but we need insert some cells in the some context, but because it happens in a different context, the application crashed